src/transformers/generation/utils.py

```diff
@@ -2200,6 +2200,25 @@ def _tensor_or_none(token, device=None):
         generation_config._pad_token_tensor = pad_token_tensor
         generation_config._decoder_start_token_tensor = decoder_start_token_tensor
 
+    def _is_dynamo_compilation_disabled(self):
+        """
+        Check if dynamo compilation should be disabled based on environment variables.
+        Uses standard environment variables to respect user's explicit intention to disable
+        torch.dynamo compilation.
+        """
+        disable_env_vars = [
+            "ACCELERATE_DISABLE_TORCH_DYNAMO",  # Accelerate-specific
+            "TORCH_DYNAMO_DISABLE",  # Pytorch standard
+            "DISABLE_TORCH_DYNAMO",  # Common alternative
```

Comment:
This is not a standard env var, correct? If not, then let's remove it 🤗 (let's stick to the known flags and avoid adding more noise, which may have unintended consequences)